test/tools/ossfuzz/protoToYul.cpp

```diff
@@ -23,13 +23,51 @@
 using namespace std;
 using namespace yul::test::yul_fuzzer;
 
+std::string const yul::test::yul_fuzzer::createHex(std::string const& _hexBytes)
```

Comment:
There's `using namespace std` above, no need to prefix these.

Reply:
Yeah, this was suggested before. There is a problem with the compiler toolchain that the fuzzer relies on. tl;dr sadly we need to keep the `std::` prefix here.

Reply:
Do you mean that `string` also exists in another imported namespace in the case of the fuzzer toolchain?

Reply:
@axic Yes, the fuzzing binary is linked against libc++ and the `std::` here ensures we link against the legacy (libstdc++) implementation and not libc++.

I don't know why we have competing implementations of the standard library though :man_shrugging: